en/includes/guides/identity-verification/manage-identity-verification-provider.md

```diff
@@ -0,0 +1,60 @@
+## Manage an Identity Verification Provider
+
+This section provides instructions on how to manage Identity Verification Providers (IDVPs) once they are created.
+
+### View an Identity Verification Provider
+
+To view the list of IDVPs added to the {{ product_name }}:
+
+1. On the {{ product_name }} Console, go to **Connections**.
+
+2. Select **Identity-Verification** filter tag to filter out the created IDVPs.
+   ![idvp-list]({{base_path}}/assets/img/guides/identity-verification/view-idvp.png)
+
+### Edit an Identity Verification Provider
+
+An administrator can update an IDVP via the {{ product_name }} Console.
+
+To edit an IDVP:
+
+1. On the {{ product_name }} Console, go to **Connections**.
+
+2. Find the IDVP and click **Setup up**.
```

Comment:
```suggestion
2. Find the IDVP and click **Setup**.
```

Reply:
In the UI, this is mentioned as `Setup up`. Therefore keeping as it is in the documentation.

Reply:
In the UI, this is mentioned as `Set up`.
Updated: f31e6771411756676f904556a8900e19e6acce6a